libs/isograph-react/src/core/read.ts

```diff
@@ -636,7 +643,11 @@ function generateChildVariableMap(
   const childVars: Writable<Variables> = {};
   for (const [name, value] of fieldArguments) {
     if (value.kind === 'Variable') {
-      childVars[name] = variables[value.name];
+      const variable = variables[value.name];
+      if (variable == null) {
+        throw new Error('Variable ' + value.name + ' is not missing');
```

Comment:
s/not missing/missing

But also, this breaks if we have a default value for a variable and the variable is not provided, e.g. `field Query.PetDetailRoute($id: ID = 4) @component {` with no variable provided.

I think the appropriate behavior is to continue the loop, i.e. we don't (currently) distinguish between missing and null variables, so skipping the variable is appropriate (and based on my limited testing, works)

Reply:
Oh ChatGPT, thank you for being awful lol. I did not even see the "not" lol

And that logic makes sense! Will just allow the loop to continue.